source/docs/power-and-electronics/driver-station-guide.rst

```diff
@@ -1,13 +1,53 @@
 Driver Station Guide
 ====================
 
-From many years of collective experience, the writers have accumulated some general recommendations and best practices for driver stations to reduce the risk of disconnects.
+The driver station is the communication point for drivers. It allows you to configure settings, configure hardware, start and stop opmodes, send gamepad data, and can receive telemetry data from the robot. There are two main different choices for a driver station, the REV Robotics Driver Hub and an Android phone.
```

Comment:
```suggestion
```suggestion
The Driver Station (DS) is the way drivers interact with the robot. Through it, you configure settings, configure hardware, start and stop OpModes, use gamepads, and receive telemetry data from the robot. There are two main different choices for a Driver Station: the REV Robotics Driver Hub and an Android phone.
```
```